src/layout/Group/types.d.ts

```diff
@@ -29,6 +29,7 @@ export interface IGroupEditProperties {
 export interface ILayoutGroup extends ILayoutCompBase<'Group'> {
   children: string[];
   maxCount?: number;
+  minCount?: number;
```

Comment:
You should add this to `layout.schema.v1.json` as well. 🙏

Reply:
Definetely! Oversight on my part 🤓